k8s/policies.go

```diff
@@ -0,0 +1,238 @@
+package k8s
+
+import (
+	"reflect"
+	"strings"
+
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/yaml"
+	"k8s.io/client-go/kubernetes"
+	typedCorev1 "k8s.io/client-go/kubernetes/typed/core/v1"
+)
+
+const PolicyAnnotationKey = "com.openfaas.policies"
+
+// PolicyClient defines the interface for CRUD operations on policies
+// and applying faas-netes policies to function Deployments.
+type PolicyClient interface {
+	Get(namespace string, names ...string) ([]Policy, error)
+}
+
+// Policy is and openfaas api extensions that can be predefined and applied
+// to functions by annotating them with `com.openfaas/policy: name1,name2`
```

Comment:
I think I prefer `com.openfaas/policy: name1,name2`

Reply:
this was updated